### PR TITLE
Fix TableSimple styles

### PR DESCRIPTION
**Pull request by alxnddr**
### Description & Demo

TableSimple is the table we render on dashboards.

**1. Fixes the position of the sorting icon:**

1.1 Left aligned column
Screenshot 2023-11-17 at 5 19 48 PM

1.2 Right aligned column
Screenshot 2023-11-17 at 5 19 53 PM

**2. Show PK and FK as pills**
Screenshot 2023-11-17 at 5 19 48 PM

### How to verify

Describe the steps to verify that the changes are working as expected.

1. New question -> Sample Dataset -> Orders -> Save the table as a question
2. Add the question to a dashboard
3. Ensure the sorting icon looks as expected
4. Ensure PK and FK columns show values as "pills"

### Checklist

- [n/a] ~Tests have been added/updated to cover changes in this PR~ visual change

### CODE REVIEW

**Review by mazameli — COMMENTED** [review]
This is a great fix, thanks. 

One opportunistic thing I still think we could do would be to increase the opacity of the chevron in the default state, like here: I can barely see this at all. Do you know what the percent value is for the opacity?

image

**Comment by alxnddr** [comment]
@mazameli good idea! It was 0.2 and I bumped it to 0.5:
Screenshot 2023-11-20 at 1 54 17 PM

How does this look to you?

**Comment by mazameli** [comment]
@alxnddr let me take a real quick look in the branch — it's sometimes hard to assess this from a screenshot.

**Review by mazameli — APPROVED** [review]
Lovely, thank you!

File: frontend/src/metabase/visualizations/components/TableSimple/TableSimple.styled.tsx [hunk 1]
```diff
@@ -71,19 +72,20 @@ export const TableHeaderCellContent = styled.button<{
...
   display: flex;
-  justify-content: ${props =>
-    props.isRightAligned ? "space-between" : "flex-start"};
+  align-items: center;
+  justify-content: flex-start;
   width: 100%;
+  flex-direction: ${props => (props.isRightAligned ? "row-reverse" : "row")};
 
-  margin-left: ${props => (props.isRightAligned ? "auto" : "unset")};
+  //margin-left: ${props => (props.isRightAligned ? "auto" : "unset")};
```
Comment (kulyk) [inline frontend/src/metabase/visualizations/components/TableSimple/TableSimple.styled.tsx:80]: Should we just remove this line?

**Review by kulyk — APPROVED** [review]
LGTM, please take a look if we should remove that commented out line

File: frontend/src/metabase/visualizations/components/TableSimple/TableSimple.styled.tsx (hunk 1 above)
Comment (alxnddr) [reply frontend/src/metabase/visualizations/components/TableSimple/TableSimple.styled.tsx:80]: Yes, I forgot it, thanks for pointing out

**Pull request merged**